src/pages/dashboard/_MapContainer.js

```diff
@@ -146,7 +146,7 @@ const MapContainer = () => {
         tabIndex={0}
       >
         <span className={useChoropleth ? '' : 'active'}>Bubble Map</span>
-        <span className={useChoropleth ? 'active' : ''}>Choropleth</span>
+        <span className={useChoropleth ? 'active' : ''}>Choropleth Map</span>
```

Comment:
Good catch, we're gonna use a better word here than "choropleth" since most folks have no idea what that is